### ISSUE TITLE
Help me understand...

### ISSUE THREAD

**Issue by bensquire** [duncan3dc/sonos#20]
Great library, thanks for all the effort!

Quick question. I'm trying to share music from my Mac, i can get it to take music thats served over http (like this http://www.tonycuffe.com/mp3/tail%20toddle.mp3). However I want to share using SMB/CIFS as well?

All I've been able to achieve so far are (uPNP) 701 errors. Below is a sample of my code:

```
$directory = new \duncan3dc\Sonos\Directory('/Users/ben/Music', '192.168.1.96/Music', '');
$text = new \duncan3dc\Sonos\Tracks\TextToSpeech('Welcome.', $directory);
$controller->interupt($text, 5);
```

and the error I receive:

```
PHP Fatal error:  Uncaught exception 'duncan3dc\Sonos\Exceptions\SoapException' with message 'UPnPError: 701' in /Users/ben/www/sonos/vendor/duncan3dc/sonos/src/Device.php:121
Stack trace:
#0 /Users/ben/www/sonos/vendor/duncan3dc/sonos/src/Speaker.php(91): duncan3dc\Sonos\Device->soap('AVTransport', 'Play', Array)
#1 /Users/ben/www/sonos/vendor/duncan3dc/sonos/src/Controller.php(193): duncan3dc\Sonos\Speaker->soap('AVTransport', 'Play', Array)
#2 /Users/ben/www/sonos/vendor/duncan3dc/sonos/src/Controller.php(675): duncan3dc\Sonos\Controller->play()
#3 /Users/ben/www/sonos/test.php(28): duncan3dc\Sonos\Controller->interupt(Object(duncan3dc\Sonos\Tracks\TextToSpeech), 5)
#4 {main} thrown in /Users/ben/www/sonos/vendor/duncan3dc/sonos/src/Device.php on line 121
```

![screen shot 2015-04-25 at 00 31 16](https://cloud.githubusercontent.com/assets/600237/7330234/69a139a8-eae2-11e4-8a9e-1ce55c849320.png)

Any help would be greatly appreciated :)
Ben

**Comment by duncan3dc** [comment]
Hey Ben, thanks, glad you're finding it useful.

I presume your cifs share works from the official Sonos app?

Could you try turning logging on and posting the result in a gist?
You can do this using `monolog` like so:
```php
$logger = new \Monolog\Logger("sonos");
$logger->pushHandler(new \Monolog\Handler\StreamHandler("/tmp/sonos.log", \Monolog\Logger::DEBUG));
$sonos->setLogger($logger);
```

**Comment by bensquire** [comment]
Thanks @duncan3dc ,

As it happens you've indirectly given me the answer to the question... By adding the folder I wanted to share into Sonos, I was able to see the CIFS share name, which when I plugged into my code.

Et Voila!

Can I suggest something is added into the documentation around folder sharing and streaming as this would make the already excellent documentation, practically perfect.

Thanks once again,
Ben


````
$controller = $sonos->getControllerByRoom("Kitchen");
$directory = new \duncan3dc\Sonos\Directory('/Users/ben/Music', 'MACBOOKPRO/Music', '');
$text = new \duncan3dc\Sonos\Tracks\TextToSpeech('Hello Dave', $directory);
$controller->interupt($text, 25);
